test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/endpointproviders/OperationContextParametersTest.java

```diff
@@ -56,9 +64,36 @@ public void setup() {
             .thenThrow(new RuntimeException("boom"));
     }
 
-    @ParameterizedTest
-    @MethodSource("testCases")
-    void operationContextParams_customization_resolvedCorrectly(TestCase tc) {
+    static Stream<Arguments> paramAssertions() {
+        return Stream.of(
+            Arguments.of("Single String parameter works as expected", new TestAssertion(
+                p -> assertThat(p.pojoString()).isEqualTo("StringMemberA"))),
+
+            Arguments.of("List of String parameter works as expected", new TestAssertion(
+                p -> assertThat(p.basicListOfString()).isNotEmpty().hasSize(2).containsExactly("StringA", "StringB"))),
+
+            Arguments.of("List of String parameter from wildcard works as expected", new TestAssertion(
+                p -> assertThat(p.wildcardKeyListOfString()).isNotEmpty().hasSize(2)
+                                                            .containsExactly("StringMemberS1", "StringMemberS2"))),
```

Comment:
Can we write a test case when only one of the List has entry
 
 Source Json
 ```json
{
  "bucket": "someBucket",
  "delete": {
    "objects": [
      {
        "id": "a",
        "key": "One"
      },
      {
        "id": "b"
      }
    ]
  }
}

```
JMESPath Expression : `delete.objects[*].key`

Output
```json
[
  "One"
]
```

Reply:
Yes, had to update the test harness to handle this case.